Assets/Scripts/trial/FieldTrial.cs

```diff
@@ -19,13 +20,26 @@ public FieldTrial(InputField[] fields) : base(-1, -1)
         {
             _fields = fields;
 
-            // This block of code basically calls the directory picker to select the configuration file.
-            while (true)
+            // This block of code will use the default configuration file if
+            // using webGL or Android (can add others in if statement) 
+            // Otherwise calls the directory picker to select the configuration file.
+
+            string[] files = Directory.GetFiles(Application.streamingAssetsPath + "/Default_Config/");
+            Debug.Log(files);
+            if (files.Length > 0)
+            {
+                string defaultConfig = Application.streamingAssetsPath + "/Default_Config/" + Path.GetFileName(files[0]);
+                Loader.ExternalActivation(defaultConfig);
+            }
+            else
             {
-                // Here we're using: https://github.com/gkngkc/UnityStandaloneFileBrowser because it was easier than rolling our own
-                string[] paths = StandaloneFileBrowser.OpenFilePanel("Choose configuration file", "Configuration_Files", "", false);
-                string path = paths[0];
-                if (Loader.ExternalActivation(path)) break;
+                while (true)
+                {
+                    // Here we're using: https://github.com/gkngkc/UnityStandaloneFileBrowser because it was easier than rolling our own
+                    string[] paths = StandaloneFileBrowser.OpenFilePanel("Choose configuration file", "Configuration_Files", "", false);
```

Comment:
Best to change these two locations as well

Reply:
Change locations so that all the configuration files are in one place?

Reply:
I want to keep Default configuration file separated from other configuration files for the time being so that i don't have to redo tutorial videos and so that the dafault configuration file will automatically get built into the application via streaming assets.